python/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM python:3.9-slim-buster
+WORKDIR /app
+COPY ./ ./
```

Comment:
copy specific file (in this case just one..) to avoid confusion and future problems